android/assets/shotIndicators/shotIndicators.atlas

```diff
@@ -0,0 +1,13 @@
+
+shotIndicators.png
+size: 128,64
+format: RGBA8888
+filter: Nearest,Nearest
+repeat: none
+Excalibur
```

Comment:
You use "Excalibur" some places but "Filled white shot indicator" other places. If this is going to be implemented as a setting consistency would save us a headache later-on